Bootstrap/Setup.php

```diff
@@ -130,6 +130,7 @@ private function createTables()
               `articleID` INT(11) NOT NULL DEFAULT '0',
               `articledetailsID` INT(11) NOT NULL DEFAULT '0',
               `price` DOUBLE DEFAULT '0',
+              `pseudoPrice` DOUBLE DEFAULT '0',
```

Comment:
Changes done. Hope they are okay. I have forked the repository today, so I can not see that my branch is out-dated.